assets.md

```diff
@@ -358,6 +362,61 @@ The OpenTelemetry Bot addresses two common issues:
 
    [Personal Access Token]: https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token
 
+### otelbot
+
+This is a GitHub App owned by [@open-telemetry](https://github.com/open-telemetry) that you can use when
+automating common GitHub tasks in OpenTelemetry repos (e.g. release automation tasks).
+
+This GitHub App has the following permissions:
+
+- Read access to metadata
+- Read and write access to pull requests
+
+- Admins: [@open-telemetry/admins](https://github.com/orgs/open-telemetry/teams/admins
+
+This GitHub App addresses two common issues:
+
+1. Since you can't push directly to `main` from workflows (due to branch protections), the next best thing is to
+   generate a pull request from the automation and use an account which has signed the CLA as the commit author.
+
+   The OpenTelemetry Bot account has signed the CNCF CLA, and you can assign it as the commit author in your automation:
+
+   ```
+   git config user.name otelbot
+   git config user.email 197425009+otelbot@users.noreply.github.com
+   ```
+
+   It is recommended to push to branch names that start with `otelbot/`, and to add a branch protection
+   rule for `otelbot/**/**` with the same setup as documented for
+   [`dependabot/**/**`](docs/how-to-configure-new-repository.md#branch-protection-rule-dependabot). Note that branch protection rule ordering matters, so you will need to
+   delete the `**/**` branch protection rule temporarily, then add the `otelbot/**/**` branch protection
+   rule, then add back the `**/**` branch protection rule.
```

Comment:
Callout the cautionary note:

```suggestion
   [`dependabot/**/**`](docs/how-to-configure-new-repository.md#branch-protection-rule-dependabot).
   
   > [!CAUTION]
   > Branch protection rule **ordering** matters, so you will need to delete the `**/**` branch protection rule temporarily, then add the `otelbot/**/**` branch protection rule, then add back the `**/**` branch protection rule.
```

Not sure if the above is properly formatted, but you get the idea.